Formula/cassandra.rb

```diff
@@ -136,6 +116,7 @@ def install
     bin.write_exec_script Dir["#{libexec}/bin/*"]
 
     rm %W[#{bin}/cqlsh #{bin}/cqlsh.py] # Remove existing exec scripts
+    pypath = libexec/"vendor"/Language::Python.site_packages("python3")
     (bin/"cqlsh").write_env_script libexec/"bin/cqlsh", PYTHONPATH: pypath
     (bin/"cqlsh.py").write_env_script libexec/"bin/cqlsh.py", PYTHONPATH: pypath
```

Comment:
I _think_ this should fix it. Also, the `rm` call is probably redundant.
```suggestion
    venv_python = libexec/"vendor/bin/python"
    (bin/"cqlsh").write_env_script libexec/"bin/cqlsh", CQLSH_PYTHON: venv_python
    rewrite_shebang python_shebang_rewrite_info(venv_python), libexec/"bin/cqlsh.py"
```
We need to make sure that the Python interpreter being used is the one in our `venv`. We'll need to do
```ruby
include Language::Python::Shebang
```
too.

Reply:
Nope, `CQLSH_PYTHON` doesn't behave the way I expect it to. Upon inspection, it seems to be discarded immediately after checking its existence, which is strange. Tried something else.

Reply:
Hmmm... Haven't managed to persuade this to use the `venv` python. Weird.